README.md

```diff
@@ -161,6 +173,10 @@ helm install squid ./squid --set environment=dev
 # Verify deployment (or use: mage squidHelm:status)
 kubectl get pods -n proxy
 kubectl get svc -n proxy
+
+# Verify both exporters are running
+kubectl logs -n proxy deployment/squid -c squid-exporter  # Standard metrics
+kubectl logs -n proxy deployment/squid -c squid           # Squid + integrated per-site metrics
```

Comment:
This doesn't look correct. We have 3 containers. The `squid` container doesn't integrate the per-site metrics exporter.